impl/src/test/groovy/com/okta/authn/sdk/impl/client/WrappedAuthenticationClient.groovy

```diff
@@ -15,29 +15,21 @@
  */
 package com.okta.authn.sdk.impl.client
 
-import com.okta.sdk.authc.credentials.ClientCredentials
+import com.okta.commons.http.HttpException
+import com.okta.commons.http.Request
+import com.okta.commons.http.RequestExecutor
+import com.okta.commons.http.Response
+import com.okta.commons.http.config.BaseUrlResolver
 import com.okta.sdk.cache.CacheManager
-import com.okta.sdk.client.AuthenticationScheme
-import com.okta.sdk.client.Proxy
 import com.okta.sdk.impl.api.ClientCredentialsResolver
 import com.okta.sdk.impl.config.ClientConfiguration
 import com.okta.sdk.impl.ds.InternalDataStore
-import com.okta.sdk.impl.http.Request
-import com.okta.sdk.impl.http.RequestExecutor
-import com.okta.sdk.impl.http.Response
-import com.okta.sdk.impl.http.RestException
-import com.okta.sdk.impl.http.authc.RequestAuthenticatorFactory
-import com.okta.sdk.impl.util.BaseUrlResolver
 
 class WrappedAuthenticationClient extends DefaultAuthenticationClient {
```

Comment:
I don't remember why I needed to create this `WrappedAuthenticationClient`. It may be possible to remove this now that all of the deprecated bits are gone (not %100 sure though, and this is an implementation detail so 🤷 )

Reply:
It is being used in tests. I'll leave it in as is.